config/initializers/settings_normalizer.rb

```diff
@@ -36,4 +37,14 @@ def normalize_path(path)
 
     Rails.root.join(path)
   end
+
+  def normalize_worker_groups
+    groups = @settings.sneakers
+    groups.map do |group|
+      classes = group.classes
+      classes = [classes] unless classes.is_a?(Array)
+      group.classes = classes
```

Comment:
You can simply convert it to an array by replacing the three lines above with
```ruby
group.classes = Array(group.classes)
```

Reply:
Yea, but that would be too easy.